components/brave_ads/core/internal/ads/serving/choose/sample_ads.h

```diff
@@ -53,6 +61,20 @@ absl::optional<T> SampleAdFromPredictors(
   return absl::nullopt;
 }
 
+template <typename T>
+std::vector<double> ComputeProbabilities(const std::vector<T>& scores) {
```

Comment:
will we have different types for `scores`? maybe it would be better to use only `std::vector<int>` type here?

Reply:
Will we have any other types for scores rather than vector<int> ?

Reply:
it should always be a vector. Or at least it should be vectors for this base/primitive implementation

Reply:
can we have other types for scores then vector<int>?